styles.css

```diff
@@ -15,8 +15,9 @@ table {
     A border-collapse value of collapse is standard best practice for any table styling effort.
     By default, when you set borders on table elements, they will look pretty ugly.
      */
+    
     border-collapse: collapse;
-    border: 3px solid purple;
+    border: 3px solid lightblue;
 }
 
```

Comment:
The changes in this PR look great! I have a couple of suggestions:

1. On line 19, it looks like the indentation is off. Please make sure it aligns with the previous line.
2. On line 22, the color 'lightblue' might not provide enough contrast with the text. Consider using a darker color to improve readability.

Overall, great work!